.github/workflows/chaos-experiment.yaml

```diff
@@ -0,0 +1,85 @@
+---
+name: chaos testing for kubi with Litmus
+on:
+  - push
+jobs:
+  kind:
+    runs-on: ubuntu-latest
+    name: create cluster KinD  with (1.21,1.23) Kub version
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes:
+          - 1.21
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: Create cluster KinD
+        uses: helm/kind-action@v1.4.0
+        with:
+          config: .github/kind-cluster-${{ matrix.kubernetes }}.yaml
+      - name: testing cluster kinD
+        run: kubectl cluster-info --context kind-chart-testing echo " current-context:"
+          $(kubectl config current-context)
+                 
+      - name: Setting up kubeconfig ENV for Github Chaos Action
+        run: echo ::set-env name=KUBE_CONFIG_DATA::$(base64 -w 0 ~/.kube/config)
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+          go-path: ${{ github.workspace }}/go
+          
+      - name: Set GOROOT
+        run: echo "export GOROOT=/opt/hostedtoolcache/go/1.17.13/x64" >> $GITHUB_ENV 
+          
+      - name: Create certificate for kubi
```

Comment:
For a test, maybe you can just ship a default set of certificates and values, so that the test is relatively cheaper, faster to load, and won't need to fetch stuff from the external world? This would make the life simpler.